docs/adding_packages/sources_and_patches.md

```diff
@@ -86,8 +88,7 @@ reading the changelog of the library, the documentation, or any statement by the
 
 Patch files are preferred over programmatic `replace_in_file` statements. This makes it easier to review and prevent
 unwanted side effects when new versions are added. They will be listed in [`conandata.yml`](conandata_yml_format.md)
-file and exported together with the recipe. Patches must always include [patch fields](conandata_yml_format.md#patches-fields)
-which are enforced by the [linters](../../linter/conandata_yaml_linter.py).
+file and exported together with the recipe. Patches must always include [patch fields](conandata_yml_format.md#patches-fields).
```

Comment:
the only required field is the path to the patch file - everything else is optional :D

Reply:
Addressed by 75461cc13016357ac2b7a51d7e2c5b060afa05af